notebooks/visualization/python-awips-WorkingWithUpperAirObs.ipynb

```diff
@@ -0,0 +1,333 @@
+    "\n",
+    "---\n",
+    "\n",
+    "## 5. Draw the Plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create SkewT/LogP\n",
+    "plt.rcParams['figure.figsize'] = (10, 12)\n",
+    "skew = SkewT()\n",
+    "skew.plot(p, T, 'r', linewidth=2)\n",
+    "skew.plot(p, Td, 'g', linewidth=2)\n",
+    "skew.plot_barbs(wpres, u, v)\n",
+    "skew.ax.set_ylim(1000, 100)\n",
+    "skew.ax.set_xlim(-60, 30)\n",
+    "\n",
+    "title_string = \" T(F)   Td   \" \n",
+    "title_string +=  \" \" + str(ob.getString(\"staName\"))\n",
+    "title_string += \" \" + str(ob.getDataTime().getRefTime())\n",
+    "title_string += \" (\" + str(ob.getNumber(\"staElev\")) + \"m elev)\"\n",
+    "title_string += \"\\n\" + str(round(T[0].to('degF').item(),1))\n",
+    "title_string += \"  \" + str(round(Td[0].to('degF').item(),1))\n",
+    "plt.title(title_string, loc='left')\n",
+    "\n",
+    "# Calculate LCL height and plot as black dot\n",
+    "lcl_pressure, lcl_temperature = lcl(p[0], T[0], Td[0])\n",
+    "skew.plot(lcl_pressure, lcl_temperature, 'ko', markerfacecolor='black')\n",
+    "\n",
+    "# Calculate full parcel profile and add to plot as black line\n",
+    "prof = parcel_profile(p, T[0], Td[0]).to('degC')\n",
+    "skew.plot(p, prof, 'k', linewidth=2)\n",
+    "\n",
+    "# An example of a slanted line at constant T -- in this case the 0 isotherm\n",
+    "l = skew.ax.axvline(0, color='c', linestyle='--', linewidth=2)\n",
+    "\n",
+    "# Draw hodograph\n",
+    "ax_hod = inset_axes(skew.ax, '30%', '30%', loc=3)\n",
+    "h = Hodograph(ax_hod, component_range=max(wsMan))\n",
+    "h.add_grid(increment=20)\n",
+    "h.plot_colormapped(u, v, spd)\n",
+    "\n",
+    "# Show the plot\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## See also\n",
+    "\n",
+    "Documentation for:\n",
+    "\n",
+    "* [awips.DataAccessLayer](http://unidata.github.io/python-awips/api/DataAccessLayer.html)\n",
+    "* [awips.PyGeometryData](http://unidata.github.io/python-awips/api/PyGeometryData.html)\n",
+    "* [matplotlib.pyplot](https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.pyplot.html)\n",
+    "* [metpy.plots.SkewT](https://unidata.github.io/MetPy/latest/api/generated/metpy.plots.SkewT.html)\n",
+    "* [metpy.plots.Hodograph](https://unidata.github.io/MetPy/latest/api/generated/metpy.plots.Hodograph.html)\n",
+    "\n",
+    "### Related Notebooks:\n",
+    "\n",
+    "* [python-awips: How to Access Data](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/dataAccess/python-awips-HowToAccessData.ipynb)\n",
+    "\n",
+    "\n",
+    "<a href=\"#top\">Top</a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```

Comment:
You have one blank cell at the end here!

Reply:
Okay, I pushed a change for the blank cell.  That should be good once the checks all finish.